### DISCUSSION TITLE
TSX: how to fix `Could not find a declaration file for module '@tabler/icons'`?

### DISCUSSION THREAD

**Discussion by uyriq** [tabler/tabler-icons#371] [Q&A] [answered]
Is there any better solution then an 
empty file `src\tabler__icons.d.ts` ?
```Typescript
declare module '@tabler/icons'
```

**Comment by woody146** [comment]
Don't set 
```
"moduleResolution": "node16"
```

**Reply by Shakeskeyboarde** [reply]
It's a bug: https://github.com/tabler/tabler-icons/issues/424. The package "exports" config is incorrect.

**Comment by codecalm** [comment, marked as answer]
you should use `@tabler/icons-react` - React version has been moved to new package. All typescript bugs has been resolved :)

**Reply by uyriq** [reply]
i will try it, thanks
